certbot-dns-rfc2136/certbot_dns_rfc2136/__init__.py

```diff
@@ -21,15 +21,18 @@
 -----------
 
 Use of this plugin requires a configuration file containing the target DNS
-server that supports RFC 2136 Dynamic Updates, the name of the TSIG key, the
-TSIG key secret itself and the algorithm used if it's different to HMAC-MD5.
+server and optional port that supports RFC 2136 Dynamic Updates, the name
+of the TSIG key, the TSIG key secret itself and the algorithm used if it's
+different to HMAC-MD5.
 
 .. code-block:: ini
    :name: credentials.ini
    :caption: Example credentials file:
 
    # Target DNS server
    dns_rfc2136_server = 192.0.2.1
+   # Target DNS port
```

Comment:
I would probably add a note that this is optional here.

Reply:
It does say it's optional in the little text above.
dns_rfc2136_algorithm is optional too, do you want a note there too?

Reply:
Oh, you are correct. No need to add it to the comment.